lib/samples/find_route/find_route_sample.dart

```diff
@@ -25,20 +25,23 @@ class FindRouteSample extends StatefulWidget {
 }
 
 class _FindRouteSampleState extends State<FindRouteSample> {
+  // the map view controller.
   final _mapViewController = ArcGISMapView.createController();
+  // the graphics overlay for the stops.
   final _stopsGraphicsOverlay = GraphicsOverlay();
+  // the graphics overlay for the route.
   final _routeGraphicsOverlay = GraphicsOverlay();
-  final _routeLineSymbol = SimpleLineSymbol(
-      style: SimpleLineSymbolStyle.solid, color: Colors.blue, width: 5.0);
-  final _routeStartSymbol = SimpleMarkerSymbol(
-      style: SimpleMarkerSymbolStyle.circle, color: Colors.green, size: 8.0);
-  final _routeEndSymbol = SimpleMarkerSymbol(
-      style: SimpleMarkerSymbolStyle.circle, color: Colors.red, size: 8.0);
-  final List<Stop> _stops = [];
+  // the stops for the route.
+  final _stops = <Stop>[];
+  // whether the route has been generated.
   var _isRouteGenerated = false;
+  // whether the map is ready.
```

Comment:
Also for the flag relating to UI that we set in `onMapViewReady`, I think we should try be consistent across the samples so that it is clear what that flag relates to. We've used `_ready` elsewhere. I lean towards that unless we feel `_isReady` would be more 'correct' with being a bool. Any thoughts @PaulAllanSturm ?

Reply:
I like `_ready`. I feel like Dart is more informal so we shouldn't have a naming convention that booleans have `is` in the name. In which case `_isRouteGenerated` should be `_routeGenerated` as well.

Reply:
Sounds good to me 👍